js/RTCPeerConnection.js

```diff
@@ -584,6 +587,84 @@ RTCPeerConnection.prototype.createDataChannel = function (label, options) {
 	return new RTCDataChannel(this, label, options);
 };
 
+RTCPeerConnection.prototype.getStats = function () {
+	var self = this,
+		isPromise,
```

Comment:
Most of the APIs in RTCPeerConnection.js are using promises already (look at setLocalDescription for example), what do you mean by being inconsistent?